src/layout/elkLayout.ts

```diff
@@ -5,24 +5,182 @@ import { formatText, measureText } from './utils';
+   * @see https://www.eclipse.org/elk/reference/options/org-eclipse-elk-edgeRouting.html
+   */
   'org.eclipse.elk.edgeRouting': 'ORTHOGONAL',
+
+  /**
+   * Adds bend points even if an edge does not change direction.
+   * If true, each long edge dummy will contribute a bend point to its edges
+   * and hierarchy-crossing edges will always get a bend point where they cross hierarchy boundaries.
+   * By default, bend points are only added where an edge changes direction.
+   *
+   * @see https://www.eclipse.org/elk/reference/options/org-eclipse-elk-layered-unnecessaryBendpoints.html
+   */
   'elk.layered.unnecessaryBendpoints': 'true',
+
+  /**
+   * The spacing to be preserved between nodes and edges that are routed next to the node’s layer.
+   * For the spacing between nodes and edges that cross the node’s layer ‘spacing.edgeNode’ is used.
+   *
+   * @see https://www.eclipse.org/elk/reference/options/org-eclipse-elk-layered-spacing-edgeNodeBetweenLayers.html
+   */
   'elk.layered.spacing.edgeNodeBetweenLayers': '50',
+
+  /**
+   * Tells the BK node placer to use a certain alignment (out of its four)
+   * instead of the one producing the smallest height, or the combination of all four.
+   *
+   * @see https://www.eclipse.org/elk/reference/options/org-eclipse-elk-layered-nodePlacement-bk-fixedAlignment.html
+   */
   'org.eclipse.elk.layered.nodePlacement.bk.fixedAlignment': 'BALANCED',
+
+  /**
+   * Strategy for cycle breaking.
+   *
+   * Cycle breaking looks for cycles in the graph and determines which edges to reverse to break the cycles.
+   * Reversed edges will end up pointing to the opposite direction of regular edges
+   * (that is, reversed edges will point left if edges usually point right).
+   *
+   * @see https://www.eclipse.org/elk/reference/options/org-eclipse-elk-layered-cycleBreaking-strategy.html
+   */
   'org.eclipse.elk.layered.cycleBreaking.strategy': 'DEPTH_FIRST',
+
+  /**
+   * Whether this node allows to route self loops inside of it instead of around it.
+   *
+   * If set to true, this will make the node a compound node if it isn’t already,
+   * and will require the layout algorithm to support compound nodes with hierarchical ports.
+   *
+   * @see https://www.eclipse.org/elk/reference/options/org-eclipse-elk-insideSelfLoops-activate.html
+   */
   'org.eclipse.elk.insideSelfLoops.activate': 'true',
+
+  /**
+   * Whether each connected component should be processed separately.
+   *
+   * @see https://www.eclipse.org/elk/reference/options/org-eclipse-elk-separateConnectedComponents.html
+   */
   separateConnectedComponents: 'false',
+
+  /**
+   * Spacing to be preserved between pairs of connected components.
+   * This option is only relevant if ‘separateConnectedComponents’ is activated.
+   *
+   * @see https://www.eclipse.org/elk/reference/options/org-eclipse-elk-spacing-componentComponent.html
+   */
   'spacing.componentComponent': '70',
+
+  /**
+   * XXX NOT SURE - Should be spacing.baseValue?
```

Comment:
Can you add a `TODO:` or `NOTE:` comment - editors will pick those up automatically.

Reply:
You want to leave it there, and simply change it to a "TODO" if I understand you well?

Reply:
Yes plz